relationship_server.py

```diff
@@ -11,6 +11,41 @@
 nlp = spacy.load("en_core_web_lg")
 
 logger = logging.getLogger(__name__)
+
+
+'''
```

Comment:
If we can move function docstrings inside the function, they're easier to lookup and read. 

Additionally, please [black](https://github.com/psf/black) your code?

Reply:
Considered & done via 4164190571431e1a98599a07c209883d1b8c0eeb